### DISCUSSION TITLE
My contribute count automatically reducing by 100/day

### DISCUSSION THREAD

**Discussion by NishatFariza** [community/community#37092] [Other Feature Feedback, Questions, & Ideas]
Today I had 1780 contributions on my profile but after 3/4 hours I noticed my contribution count reduced to 1670.

N.B: I didn’t delete any repository or leave any organization where I have my commits.

**Comment by ossanna16** [comment]
Hi there @NishatFariza and welcome to our community! Thank you for asking a great question 🙂
